Sources/Identity/CustomerInfoManager.swift

```diff
@@ -120,45 +120,26 @@ class CustomerInfoManager {
         }
 
         do {
-            let infoDict = try JSONSerialization.jsonObject(with: customerInfoData) as? [String: Any]
-            guard let customerInfoDict = infoDict else {
-                return nil
-            }
+            let info: CustomerInfo = try JSONDecoder.default.decode(jsonData: customerInfoData)
 
-            let info: CustomerInfo
-            do {
-                info = try CustomerInfo(data: customerInfoDict)
-            } catch {
-                if let customerInfoError = error as? CustomerInfoError {
-                    Logger.error(customerInfoError.description)
-                } else {
-                    Logger.error("Error loading customer info from cache: \(error)")
-                }
-                return nil
-            }
-
-            if let schema = info.schemaVersion, schema == CustomerInfo.currentSchemaVersion {
+            if info.isInCurrentSchemaVersion {
                 return info
+            } else {
+                return nil
```

Comment:
Do we need to log anything here for this error? 🤔

Reply:
We weren't logging anything before 🤷🏻‍♂️

Reply:
Probably fine then!